skyVegasTests/gamePlayGameClient.spec.ts

```diff
@@ -1,35 +1,261 @@
-import { test, type Page } from '@playwright/test';
+import { test, type Page, expect } from '@playwright/test';
 import {
     launchGame,
     login,
-    validateConsoleMessages,
     startEventListener,
-    makeSpin,
+    firstSpin,
     readGames,
+    spin,
+    getValueFromConsoleMessages,
+    currencyStringToNumber,
 } from './utilities';
 import { ExpectedMessage } from '../types/expectedMessage';
 
 let page: Page;
 readGames().forEach((game) => {
-    const consoleMessages: string[] = [];
-
     test.describe(`Testing with text: ${game}`, () => {
-        test.beforeAll(async ({ browser }) => {
+        test.beforeEach(async ({ browser }) => {
+            test.setTimeout(300000);
             page = await browser.newPage();
             await login(page);
         });
 
-        test.afterAll(async () => {
+        test.afterEach(async () => {
             await page.close();
         });
 
-        test('gamePlay', async () => {
+        test('gamePlayForLoss', async () => {
```

Comment:
this is quite a bulky test, I can see that it's mostly using helper functions as well so might just be a big one. Will also look at tinkering with the code formatter so it doesn't add quite so many line feeds in the name of neatness